acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -1,5 +1,4 @@
 require 'test/unit/assertions'
-require 'puppetserver/acceptance/gem_utils'
```

Comment:
This appears to be in the load path already as a helper, and it is manually added as a Beaker helper in the PE acceptance script.